test/functional/helpers/spectrum.js

```diff
@@ -20,6 +20,7 @@ const popoverMenuSelector = Selector('[role="listbox"]');
 const tabSelector = Selector('[role="tab"]');
 const menuItemCssSelector = '[role="option"]';
 const invalidAttribute = "aria-invalid";
+const invalidPickerButtonClassName = "._spectrum-FieldButton--invalid_7a745";
```

Comment:
The hash on the end of this class name will be re-generated.

Reply:
I'll add a regex